src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+const counter = document.querySelectorAll('tr');
+const listName = document.querySelector('thead');
+const list = document.querySelector('tbody');
+
+const changeToNum = (value) => value.replace('$', '').replace(',', '');
+
+for (let i = 0; i < [...counter].length; i++) {
+  for (let y = 0; y < [...counter[i].children].length; y++) {
+    counter[i].children[y].count = y;
+  }
+}
+
+listName.addEventListener('click', (e) => {
+  const item = e.target;
+
+  sortedList(item.count, list);
```

Comment:
```suggestion
  sortList(item.count, list);
```
sortedList - отсортированный список
sortList - отсортировать список